README.md

```diff
@@ -117,21 +114,21 @@ That would change `Endpoint bravo` baudrate to `115200`.
 
 ### Flight stack logging ###
 
-Mavlink router can also collect flight stack log. It supports collecting
+Mavlink router can also collect flight stack logs. It supports collecting
 both PX4 and Ardupilot flight stacks logs. To start logging, set a
-directory to `Log` key on `General` section of config file (or use
+directory to the `Log` key in the `General` section in the config file (or use
 argument option `-l`).
-Currently, mavlink router needs to be informed which MAVLink dialect
-flight stack speaks, `common` or `ardupilotmega`. To define it, use
+Currently, mavlink router needs to be informed which MAVLink dialect the
+flight stack speaks, `common` or `ardupilotmega`. To define it, use the
```

Comment:
humn... this is actually outdated now. We have "auto", that should be doing the right thing.

Reply:
ah, I was wondering what auto exactly is, thx. (and I'm wondering how it actually would work). I'll update my knowledge and will revise.